deployment/ansible/roles/driver.celery/tasks/main.yml

```diff
@@ -34,7 +34,12 @@
         job="docker exec $(docker ps -q -f name=driver-celery) manage.py calculate_black_spots"
         hour=3 minute=10
 
-- name: Add remove duplicate records cronjob
-  cron: name="remove duplicate records"
-        job="docker exec $(docker ps -q -f name=driver-celery) manage.py remove_duplicates"
+- name: Add find duplicate records cronjob
+  cron: name="find duplicate records"
+        job="docker exec $(docker ps -q -f name=driver-celery) manage.py find_duplicate_records"
         hour=1 minute=10
+
+- name: Clean up CSV export files cronjob
+  cron: name="export file cleanup"
+        job="docker exec $(docker ps -q -f name=driver-celery) manage.py remove_old_exports"
+        hour=5 minute=10
```

Comment:
I think we could run this quite a bit more frequently since the command itself omits files that are less than 6 hours old. Perhaps remove the `hour` parameter so that it runs once an hour?